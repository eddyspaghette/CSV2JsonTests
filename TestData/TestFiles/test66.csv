ッ;
ツ;ぃ,ぎ,せ,℟,▓
ヅ,い,け,₠,Å,○
ミ,ぅ,げ,₡,ℬ,☝
デ,う,,₢,Δ,⭐
ト,ぇ,ご,,ψ,😵
ド,え|さ,℀,,🍕
ツ,,ぎ,せ,℟,▓
ヅ,い,け,₠,,○
ミ|ぅ,|₡,ℬ,☝
デ,う,こ,,Δ,⭐
ト,ぇ,ご,₣,ψ,😵
ド,え,さ,℀,ↂ,🍕
