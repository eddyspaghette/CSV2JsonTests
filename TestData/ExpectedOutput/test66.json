[
 {
   "ッ": "ツ",
   "FIELD2": "ぃ,ぎ,せ,℟,▓"
 },
 {
   "ッ": "ヅ,い,け,₠,Å,○",
   "FIELD2": ""
 },
 {
   "ッ": "ミ,ぅ,げ,₡,ℬ,☝",
   "FIELD2": ""
 },
 {
   "ッ": "デ,う,,₢,Δ,⭐",
   "FIELD2": ""
 },
 {
   "ッ": "ト,ぇ,ご,,ψ,😵",
   "FIELD2": ""
 },
 {
   "ッ": "ド,え|さ,℀,,🍕",
   "FIELD2": ""
 },
 {
   "ッ": "ツ,,ぎ,せ,℟,▓",
   "FIELD2": ""
 },
 {
   "ッ": "ヅ,い,け,₠,,○",
   "FIELD2": ""
 },
 {
   "ッ": "ミ|ぅ,|₡,ℬ,☝",
   "FIELD2": ""
 },
 {
   "ッ": "デ,う,こ,,Δ,⭐",
   "FIELD2": ""
 },
 {
   "ッ": "ト,ぇ,ご,₣,ψ,😵",
   "FIELD2": ""
 },
 {
   "ッ": "ド,え,さ,℀,ↂ,🍕",
   "FIELD2": ""
 }
]